package.json

```diff
@@ -83,4 +84,4 @@
     },
     "types": "dist/index",
     "version": "0.3.4"
-}
+}
```

Comment:
Mind putting this new line back & squashing this down to one commit to keep the git history a little cleaner? Saw some reverts in there

Reply:
Yep, you got it!

Reply:
@brandongregoryscott do you mean squashing when merging or do you want me to rebase it down to one commit?

Reply:
Ehh, now that I think about it, I can just squash it here. I think it'll erase your commits entirely and just show up as mine, though

Reply:
No worries, I just finished squashing it 🙂 . Should be good to go now!